test/resources/testlibs/CustomSeleniumKeywords.py

```diff
@@ -0,0 +1,18 @@
+from Selenium2Library import Selenium2Library
+from robot.utils import asserts
+from robot.libraries.BuiltIn import BuiltIn
+
+class CustomSeleniumKeywords(Selenium2Library):
+
+    def __init__(self, *args, **kwargs):
```

Comment:
OK, I've got everything in the one test file, but I had to introduce this `__init__` hack, so that the `Go To` command would work with this library.